docs/docs/configuration/theming.mdx

```diff
@@ -38,7 +38,6 @@ Set the feature flag in your `superset_config`
 ```python
 DEFAULT_FEATURE_FLAGS: dict[str, bool] = {
     {{ ... }}
-    THEME_ALLOW_THEME_EDITOR_BETA = True,
```

Comment:
No need for this feature flag now as we have an "apply" button in the theme editor in the CRUD